business_requirement_deliverable_cost/models/business.py

```diff
@@ -49,7 +49,7 @@ def _get_partner(self):
         br_id = br_deliverable = False
         if self.business_requirement_deliverable_id.id:
             br_deliverable = self.business_requirement_deliverable_id
-        if br_deliverable.business_requirement_id.id:
+        if br_deliverable and br_deliverable.business_requirement_id.id:
```

Comment:
We are using coveralls because historical reason.

But you could see the reasons to use other one here (codecov)

https://github.com/OCA/maintainer-quality-tools/pull/342